.github/workflows/actions_build.yml

```
@@ -190,3 +190,40 @@ jobs:
         run: |
           ./gradlew --no-daemon --stacktrace check -PnoLint -PflakyTests=true
         shell: bash
+
+  docker-build:
+    if: github.repository == 'line/centraldogma'
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    env:
+      GRADLE_OPTS: -Xmx1280m
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)